lib/erasure/erasure_test.go

```diff
@@ -12,100 +12,113 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestObtainChunks(t *testing.T) {
-	t.Parallel()
-	testCases := []struct {
-		name              string
-		validators        uint
-		dataHex           string
-		expectedChunksHex []string
-		expectedError     error
-	}{
-		{
-			name:              "1_validators",
-			validators:        1,
-			dataHex:           "0x04020000000000000000000000000000000000000000000000000000000000000000000000000000000000",
-			expectedChunksHex: []string{},
-			expectedError:     errors.New("expected at least 2 validators"),
+var testCases = []struct {
+	name          string
+	validators    uint
```

Comment:
nit/should this be numValidators?

Reply:
Thank you for your suggestion! 
I've actually incorporated the use of `nValidators` consistently throughout the functions in this package. As a result, I've gone ahead and updated the field name from `validators` to `nValidators` to maintain uniformity.